### PR TITLE
Adding ansys-rocky-core

### PR DESCRIPTION
**Pull request by Adrian-B-Moreira**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by igortg** [comment]
I'm Ok with being set as maintainer.

**Review by igortg — APPROVED** [review]

**Review by fingeraugusto — APPROVED** [review]

**Comment by fingeraugusto** [comment]
I'm Ok with being set as maintainer.

**Comment by Adrian-B-Moreira** [comment]
@conda-forge-admin, please ping conda-forge/help-python

File: recipes/ansys-rocky-core/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  sha256: 7ba412d28fcebc627ed8157aa9e9128573f36e9dd04021cc270b602574ffa409
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<4.0
```
Comment (ocefpaf) [inline recipes/ansys-rocky-core/meta.yaml:19]:
```suggestion
    - python {{python_min }}
```

File: recipes/ansys-rocky-core/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - flit-core >=3.2,<4
+    - pip
+  run:
+    - python >=3.9,<4.0
```
Comment (ocefpaf) [inline recipes/ansys-rocky-core/meta.yaml:23]:
```suggestion
    - python >={{python_min }}
```

File: recipes/ansys-rocky-core/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    - importlib-metadata >=4.0
+    - pyro5 >=5.13
+    - numpy >=1.19,<3.0
+
+test:
+  imports:
+    - ansys.rocky.core
+  commands:
+    - pip check
```
Comment (ocefpaf) [inline recipes/ansys-rocky-core/meta.yaml:32]:
```suggestion
    - python {{python_min }}
    - pip check
```

File: recipes/ansys-rocky-core/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  sha256: 7ba412d28fcebc627ed8157aa9e9128573f36e9dd04021cc270b602574ffa409
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >={{ python_min }},<4.0
```
Comment (ocefpaf) [inline recipes/ansys-rocky-core/meta.yaml:19]: Looks like `python_min` is not defined when linting and is empty, causing an error. BTW, you don't need the `,4`. It is very unlikely that things in this package will break and/or live that long to see breakages.

File: recipes/ansys-rocky-core/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  sha256: 7ba412d28fcebc627ed8157aa9e9128573f36e9dd04021cc270b602574ffa409
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >={{ python_min }}
```
Comment (ocefpaf) [inline recipes/ansys-rocky-core/meta.yaml:20]:
```suggestion
    - python {{ python_min }}
```

**Pull request merged**